lib/ndict.js

```diff
@@ -0,0 +1,113 @@
+(function(T) {
```

Comment:
Can you add a comment describing the purpose of this object and the url where it originates from (and it's license / creator for crediting)?